src/Controller/Organization/PackageController.php

```diff
@@ -78,6 +79,7 @@ public function packageNew(Organization $organization, Request $request, GithubA
                 return $response;
             }
         } catch (HttpException $exception) {
+            $this->dispatchMessage(new RemoveOAuthToken($this->getUser()->id()->toString(), (string) $type));
```

Comment:
I do not think this is a good idea, in the absence of a connection to the provider, we do not necessarily want to delete the token. I thought more about a separate action in the user profile.